src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/OrdinaryMethodReferenceFinder.cs

```diff
@@ -19,7 +17,8 @@ protected override bool CanFind(IMethodSymbol symbol)
                 symbol.MethodKind == MethodKind.DelegateInvoke ||
                 symbol.MethodKind == MethodKind.DeclareMethod ||
                 symbol.MethodKind == MethodKind.ReducedExtension ||
-                symbol.MethodKind == MethodKind.LocalFunction;
+                symbol.MethodKind == MethodKind.LocalFunction ||
+                symbol.MethodKind == MethodKind.Constructor;
```

Comment:
Why?  this seems wrong.  we have ConstructorSymbolReferenceFinder, why is this not there?

Reply:
Moved the logic to ConstructorSymbolReferenceFinder. Not sure how I'd missed it when I looked around.